scripts/bundleKeytar.sh

```diff
@@ -1,15 +1,13 @@
 #!/bin/bash
-# Usage: bash bundleKeytar.sh <keytarVersion> [githubAuthHeader]
```

Comment:
Same comment as before
Do you think it makes sense to add something about the ENV requirement ?